libs/prelude/Prelude.idr

```diff
@@ -214,6 +214,30 @@ Enum Char where
 
   pred c = fromNat (pred (toNat c))
 
+Enum Bits8 where
+  toNat b   = fromIntegerNat (prim__zextB8_BigInt b)
```

Comment:
Another point - Bits8/16/32/64 are defined in Prelude.idr. So if the instances are to be in the same location, they cannot depend on Data.Bits (base package).